frontend/src/components/templates/UserEnrollmentList/index.js

```diff
@@ -64,7 +64,7 @@ const UserEnrollmentList = () => {
           {showAlert && (
             <Alert
               type="success"
-              title="Votre message a bien été envoyé"
+              title="Votre habilitation a été sauvegardée"
```

Comment:
The pass might not be saved (ex: after deleting the pass). I actually removed the use of the title props during the refacto we made earlier today.

Is that ok with you to close this PR ?

Reply:
I was going to propose you to remove this PR. This one has no more need to exist.